modules/overviews/app/components/project_life_cycles/sections/edit_dialog_component.html.erb

```diff
@@ -0,0 +1,32 @@
+<%=
+  render(Primer::Alpha::Dialog.new(title: t("label_life_cycle_plural"),
+                                   size: :large,
+                                   id: "edit-project-life-cycles-dialog")) do |d|
+    d.with_header(variant: :large)
+    d.with_body(classes: "Overlay-body_autocomplete_height") do
+      render(::ProjectLifeCycles::Sections::EditComponent.new(model))
+    end
+    d.with_footer do
+      component_collection do |footer_collection|
+        footer_collection.with_component(Primer::ButtonComponent.new(
+          data: {
+            'close-dialog-id': "edit-project-life-cycles-dialog"
```

Comment:
Since this ID refers to the Dialog id defined in line 4, it would make sense to move this into a constant/method within the component. That way, if the ID changes in the future, you only need to change it in one place.

e.g.

```ruby
module ProjectLifeCycles
  module Sections
    class EditComponent
       def dialog_id
         "edit-project-life-cycles-dialog"
       end
# ...

# here:
'close-dialog-id': dialog_id
```